themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/_columns_2.html

```diff
@@ -41,7 +41,7 @@ <h3>Connect</h3>
       </ul>
     </div>
 
-    <!--<div class="sidebar-widget sidebar-widget-pulse">
+    <div class="sidebar-widget sidebar-widget-pulse">
```

Comment:
but that's ok isn't it? There is a checkbox in the page settings for showing the sidebar. I figure we might as well keep this code uncommented in case we ever do want to use it. Do you agree?